Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml

```diff
@@ -80,15 +80,10 @@
   components:
   - type: StorageFill
     contents:
-      - id: ClothingOuterHardsuitAtmos
-      - id: ClothingMaskGasAtmos
-      - id: YellowOxygenTankFilled
-      - id: NitrogenTankFilled
-      - id: ClothingOuterSuitAtmosFire
-      - id: ClothingHeadHelmetAtmosFire
```

Comment:
The firesuit isn't currently in SuitStorageAtmos

```
      - id: ClothingOuterSuitAtmosFire
      - id: ClothingHeadHelmetAtmosFire
```

Reply:
ye it needs that

Reply:
Perhaps back into the locker?